pkg/pillar/cas/cas.go

```diff
@@ -0,0 +1,66 @@
+package cas
+
+import (
+	"fmt"
+	"io"
+	"time"
+)
+
+//CAS: provides methods to interact with CAS clients
+type CAS interface {
+	//Blob APIs
+	//CheckBlobExists: returns true if the blob exists.
+	CheckBlobExists(blobSha256 string) bool
```

Comment:
Let's document that the argument is of format `<algo>:<hash>` and the current only supported one is `sha256:<hash>` (which is what containerd supports... for now). Also change the arg name to just `blob` or `blobHash` (i.e. something not purely sha256)

Reply:
Done.